atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -498,27 +500,27 @@ private void removeFromBlacklistAfterResponse(CassandraServer cassandraServer) {
         }
     }
 
+    private void logTokenRingOnStartup() {
+        Multimap<Set<TokenRange>, CassandraServer> tokenRangesToServer = getTokenRangesFromAllReachableServers();
+
+        if (tokenRangesToServer == null) {
+            log.info("Could not load token ring upon start up!");
+            return;
+        }
+
+        log.info(
+                "This is the token Range as seen by init Cassandra server pool\n{}",
```

Comment:
ngl this is pretty ugly - do we really need this formatting? I know we've done it in the past?